test/unit/evaluator_spec.js

```diff
@@ -377,54 +341,58 @@ describe("evaluator", function () {
       resources.set("XObject", xobjs);
 
       const stream = new StringStream("/Res1 Do");
-      runOperatorListCheck(
+      const result = await runOperatorListCheck(
         partialEvaluator,
         stream,
-        resources,
-        function (result) {
-          expect(result.argsArray).toEqual([]);
-          expect(result.fnArray).toEqual([]);
-          done();
-        }
+        resources
       );
+      expect(result.argsArray).toEqual([]);
+      expect(result.fnArray).toEqual([]);
     });
   });
 
   describe("thread control", function () {
-    it("should abort operator list parsing", function (done) {
+    it("should abort operator list parsing", async function () {
       const stream = new StringStream("qqQQ");
       const resources = new ResourcesMock();
       const result = new OperatorList();
       const task = new WorkerTask("OperatorListAbort");
       task.terminate();
-      partialEvaluator
-        .getOperatorList({
+
+      try {
+        await partialEvaluator.getOperatorList({
           stream,
           task,
           resources,
           operatorList: result,
-        })
-        .catch(function () {
-          expect(!!result.fnArray && !!result.argsArray).toEqual(true);
-          expect(result.fnArray.length).toEqual(0);
-          done();
         });
+
+        // Shouldn't get here.
+        expect(false).toEqual(true);
+      } catch (_) {
+        expect(!!result.fnArray && !!result.argsArray).toEqual(true);
+        expect(result.fnArray.length).toEqual(0);
+      }
     });
-    it("should abort text parsing parsing", function (done) {
+
+    it("should abort text parsing parsing", async function () {
```

Comment:
Nit: Pre-existing issue, but there's probably one "parsing" too much in the test name :-)
I'm guessing that it *should* actually be:
```suggestion
    it("should abort text content parsing", async function () {
```